test/test_unit_strings.cpp

```diff
@@ -984,6 +988,23 @@ TEST(stringToUnits, invalid)
     EXPECT_FALSE(is_valid(unit_from_string("_liquid_()")));
 }
 
+TEST(stringToUnits, ParseIssues)
+{
+    auto u1 = unit_from_string("Metres");
+    EXPECT_EQ(u1, precise::m);
+
+    u1 = unit_from_string("degrees C");
+    EXPECT_EQ(u1, precise::degC);
+
+    u1 = unit_from_string("gramm");
+    EXPECT_EQ(u1, precise::g * precise::m);
+    u1 = unit_from_string("kilogramm");
+    EXPECT_EQ(u1, precise::kilo * precise::g * precise::m);
+
+    // u1 = unit_from_string("inchy");
+    // EXPECT_EQ(u1,precise::kilo*precise::g*precise::m);
```

Comment:
Commented code?

Reply:
there for testing to see what "inchy" actually did,  will be removed before merging